about.html

```diff
@@ -5,67 +5,79 @@
+        <h2>About</h2>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+      </div>
+
+<div class="about-container">
+  <div class="about-block">
     <img src="img/about-plan.png" alt="strategy">
     
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
+      <h2>Strategy</h2>
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
+      <button>Learn More</button>
+  </div>
+  <div class="about-block">
     <img src="img/about-working.png" alt="strategy">
     
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
+      <h2>How We Work</h2>
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
+      <button>Learn More</button>
+  </div>
+  <div class="about-block">
     <img src="img/about-office.png" alt="strategy">
     
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+      <h2>Places We Work</h2>
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
+      <button>Learn More</button>
+  </div>
+  <div class="about-block">
     <img src="img/about-meeting.png" alt="strategy">
     
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      <h2>Collaboration</h2>
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+      <button>Learn More</button>
+  </div>
+  <div class="about-footer">
+    <h2>Let's Work Together</h2>
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  </div>  
+</div>
         
       <footer>
```

Comment:
`about-footer` vs `footer`? (seems confusing to me)

Reply:
I agree. Should have named it about-together, about-CTA, or something else.